website/content/en/docs/amazon-sagemaker-integration/_index.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Amazon SageMaker Integration"
+description = "Use SageMaker Operators for Kubernetes (ACK) and SageMaker Components for Kubeflow Pipelines with Kubeflow on AWS"
+weight = 40
++++
+
+Starting with Kubeflow on AWS v1.6, all of the available [deployment options]({{< ref "../deployment/_index.md" >}}) can be integrated with Amazon SageMaker using SageMaker Operators for Kubernetes (ACK) and SageMaker Components for Kubeflow Pipelines. 
+
+Amazon SageMaker is a fully managed service designed for handling performant and cost-optimized training clusters that are only provisioned when requested, scaled as needed, and shut down automatically when jobs complete, allowing you to operationalize machine leaning workflows with optimized resource utilization.
```

Comment:
Can we iterate on this in a separate PR and add more content here from the blog post

Does this page need links to respective guide so user knows what is next thing they can do